### DISCUSSION TITLE
SteamVR Overlay?

### DISCUSSION THREAD

**Discussion by uq1** [Raicuparta/rai-pal#174] [Ideas]
No hurry on this, but It would be very useful if there was an overlay in steamvr to quickly load games inside the HMD, without using desktop views.

**Comment by Raicuparta** [comment]
SteamVR itself does have an option to pin specific windows to make them float as an overlay (separate from the usual desktop view). I presume you can interact with them by opening the SteamVR overlay, but don't remember if that's really the case. Maybe worth a shot?

**Comment by Raicuparta** [comment]
I'm gonna move this to discussions since it isn't something I ever plan on doing. But yeah with SteamVR window overlay you can get pretty far. Make the Rai Pal window small (so elements are bigger in VR), then open just that specific window in the SteamVR overlay, instead of the whole desktop.
